polymath/extensions/item_ops.py

```
@@ -3,457 +3,473 @@
-    Parameters:
-        shape (tuple): The new shape for numerator items.
-        classes (class, list, or tuple, optional): The class of the object returned. If
-            a list is provided, the object will be an instance of the first suitable class
-            in the list. Otherwise, a generic Qube object will be returned.
-        recursive (bool, optional): True to reshape the derivatives in the same way;
-            otherwise, the returned object will not contain derivatives.
+        return obj
 
-    Returns:
-        Qube: The reshaped object.
+    def reshape_numer(self, shape, classes=(), recursive=True):
```

Ruff (0.14.7)

Boolean default positional argument in function definition

(FBT002)